src/styles/main.scss

```diff
@@ -1,3 +1,36 @@
+@import "./header.scss";
+@import "./product-cart.scss";
+@import "./variables.scss";
+
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.container {
```

Comment:
With BEM - you need a file per block, so `.container` styling should be in `container.scss`